.github/workflows/unpublish.yml

```diff
@@ -0,0 +1,39 @@
+name: ⛔ Deprecate
```

Comment:
File name does not match workflow name, it is better to align them, in this case i believe it is better to change workflow file to be deprecate.yml since indeed this is about deprecating and not "un-publish"